src/app/asset-page/asset-page.component.pug

```diff
@@ -19,6 +19,7 @@
               span#assetPageCount.font-weight-bold {{ totalAssetCount - restrictedAssetsCount }}
               i.icon.icon-direction.icon-right(title="Next in results", (click)="showNextAsset()", (keyup.enter)="showNextAsset()", [class.disabled]="assetNumber >= totalAssetCount - restrictedAssetsCount", [attr.tabindex]="assetNumber >= totalAssetCount - restrictedAssetsCount ? -1 : 1", aria-label="Next item, press enter to go to next result item. You can also use left and right arrow keys to navigate between result items.", role="button")
     //- for accessibility so that page has title
+    h1.sr-only([innerHTML]="getTitleAndResults()")
     h1.sr-only(*ngIf="isFullscreen") Full screen mode
```

Comment:
Would it be better if we combine those two h1s? Looks a little weird to me to have two screen-reader-only h1s on full screen mode.

Reply:
I do not have an issue with combining them, but I wonder whether the full screen mode should even be an h1? It doesn't make sense to me as a first-level heading.